flink-cyber/flink-enrichment/flink-enrichment-combined/src/main/java/com/cloudera/cyber/enrichment/EnrichmentJob.java

```diff
@@ -59,6 +63,7 @@ public abstract class EnrichmentJob {
 
     private static final String PARAMS_ENABLE_GEO = "geo.enabled";
     private static final String PARAMS_ENABLE_ASN = "asn.enabled";
+    private static final String PARAMS_ENABLE_CIDR = "cidr.enabled";
```

Comment:
Please update the flink-cyber/cyber-jobs/src/main/resources/conf/templates/triage/triage.properties.template to include the properties necessary to add cidr enrichments.  enabled should be set to false by default in the template.